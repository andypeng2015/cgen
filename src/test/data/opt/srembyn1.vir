module foo

export function w $foo(w %x) {
@start:
  %x = rem.w %x, 0xffffffff_w
  ret %x
}
