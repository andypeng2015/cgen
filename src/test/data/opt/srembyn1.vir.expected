module foo

export function w $foo(w %x) {
@0:
  ret 0x0_w
}
